src/sources/lolalytics.js

```diff
@@ -0,0 +1,304 @@
+import Promise from 'bluebird';
+import cheerio from 'cheerio';
+import R from 'ramda';
+
+import ChampionifyErrors from '../errors.js';
+import { cl, request, shorthandSkills, trinksCon } from '../helpers';
+import Log from '../logger';
+import progressbar from '../progressbar';
+import store from '../store';
+import T from '../translate';
+
+const default_schema = require('../../data/default.json');  //  Riot JSON schema
+
+//  Item headings from lolalytics website
+const item_block_names = ['Starting Items', 'First Item', 'Boots', 'Second Item', 'Third Item', 'Fourth Item', 'Fifth Item'];
+
+export const source_info = {
+  name: 'Lolalytics',
+  id: 'lolalytics'
+};
+
+export function getVersion() {
+  return request('http://current.lolalytics.com/')
+    .then(body => {
+      const $c = cheerio.load(body);
+      const lolalytics_header = $c('body > div > h2').text();
+      const patchRegex = new RegExp('League of Legends Patch ([0-9].[0-9]*)');
+      const lolalytics_ver = patchRegex.exec(lolalytics_header)[1];
+      store.set('lolalytics_ver', lolalytics_ver);
+      return lolalytics_ver;
+    })
+    .catch(err => {
+      throw new ChampionifyErrors.RequestError('Can\'t get Lolalytics Version').causedBy(err);
+    });
+}
+
+function createChampionifyJson(riot_json, position, champion_name, split_sort) {
+  if (split_sort) {
+    position += `_${split_sort}`;
+  }
+
+  return {
+    champ: champion_name,
+    file_prefix: position,
+    riot_json: riot_json,
+    source: 'lolalytics'
+  };
+}
+
+function createRiotJson(parsed_data, position, champion_name, split_sort) {
+  const blocks = [];
+  const item_names = store.get('item_names');
+  const sortByWinRate = R.sortBy(R.prop('win'));
+  const sortByPickRate = R.sortBy(R.prop('pick'));
+  const build_sections = parsed_data.build_sections;
+  const skills = parsed_data.skills;
+
+  const build_sections_to_convert = [];
+
+  R.forEach(function(x) { if (R.contains(x.title, item_block_names)) build_sections_to_convert.push(x); }, build_sections); //  Filter out the rows we don't care about (champion counters, etc.)
+
+  function convertBuildSectionJsonToBlock(build_section_json, title) {
+    const items = [];
+
+    for (let k = 0; k < build_section_json.length; k++) {
+      let id = parseInt(item_names[build_section_json[k].name], 10);
+
+      id = (isNaN(id) ? 2003 : id); //  For biscuit
+
+      items.push({
+        count: 1,
+        id: id.toString()
+      });
+    }
+
+    const filtered_items = R.uniqBy(R.prop('id'), items); //  filter out duplicates, mainly just the second health pot from converting buscuits
```

Comment:
Lolalytics doesn't track the number of items purchased. Most of the starting item suggestions included biscuit and health pot like below. The biscuit would be converted to a health pot like in other sources, but then you'd get some where health pot showed up twice. This was just to filter the second health pot out so that it didn't show up in two different spots in the builds.
![image](https://cloud.githubusercontent.com/assets/20735347/20611579/4e7c5b82-b27b-11e6-8109-e13ede3bed89.png)